### PR TITLE
105832 update new pipeline job page

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: src/components/breadcrumb/index.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
   if (link.to) {
     return (
-      <NavLink className="breadcrumb__link" to={link.to}>
-        {link.label}
-      </NavLink>
+      <>
+        <NavLink className="breadcrumb__link" to={link.to}>
+          {link.label}
+        </NavLink>
+        <span className="breadcrumb__link-space">/</span>
```
Comment (nilsgstrabo) [inline src/components/breadcrumb/index.js:14]: I would recommend using ::after css to add "/" as separator between breadcrumb items. Combine this with :not(:last-child) to exclude "/" for the last element
The way it is now, if a breadcrumb item without [to] is inserted anywhere but at the end, it won't get the "/" separator.

CSS and component example:
```css
.breadcrumb {
  font-size: 1.000rem;
}

.breadcrumb-list {
  display: inline-block;
}

.breadcrumb-list__item {
  display: inline-block;
}

:not(:last-child).breadcrumb-list__item::after {
  content: "/";
  margin: 0 var(--eds_spacing_medium) 0 var(--eds_spacing_medium);
}

.breadcrumb__text {
  color: var(--eds_text__static_icons__tertiary,rgba(111,111,111,1));
}

.breadcrumb__link {
  color: var(--eds_interactive_primary__resting);
}
```

```javascript
import PropTypes from 'prop-types';
import React from 'react';
import { NavLink } from 'react-router-dom';

import './style.css';

const BreadcrumbLink = (link) => {
  if (link.to) {
    return (
      <>
        <NavLink className="breadcrumb__link" to={link.to}>
          {link.label}
        </NavLink>
      </>
    );
  }

  return <span className="breadcrumb__text">{link.label}</span>;
};

export const Breadcrumb = ({ links }) => {
  const linksRender = links.map((link, idx) => (
    <li className="breadcrumb-list__item" key={link.to || idx}>
      {BreadcrumbLink(link)}
    </li>
  ));

  return (
    <nav className="breadcrumb" role="navigation" aria-label="Page hierarchy">
      <ul className="breadcrumb-list">{linksRender}</ul>
    </nav>
  );
};

Breadcrumb.propTypes = {
  links: PropTypes.arrayOf(
    PropTypes.shape({
      label: PropTypes.node.isRequired,
      to: PropTypes.string,
    })
  ).isRequired,
};

export default Breadcrumb;
```

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Requested changes on breadcrumb component

File: src/components/breadcrumb/index.js (hunk 1 above)
Comment (Pespiri) [reply src/components/breadcrumb/index.js:14]: I will have a look into it, but it was removed from the .css to more closely resemble the EDS component until it has been fixed

Comment (Pespiri) [reply src/components/breadcrumb/index.js]: The '/' not appearing after non-link breadcrumbs should now have been fixed

Comment (nilsgstrabo) [reply src/components/breadcrumb/index.js]: I still think it is better to use ::after to insert the "/" instead of using a <span>, the way it was before.

Comment (Pespiri) [reply src/components/breadcrumb/index.js]: Updated the .css with your example with focus on the EDS styling

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**